Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,262 @@
+
+            incrementButton
+                .accessibility(addTraits: [.isButton])
+        }
+    }
+
+    @ViewBuilder var valueContainer: some View {
+        Text("\(value)")
+            .foregroundColor(colorScheme == .light ? .cloudDark : .cloudLight)
+            .transition(
+                AnyTransition.asymmetric(
+                    insertion: .move(edge: isIncreased ? .top : .bottom),
+                    removal: .move(edge: isIncreased ? .bottom : .top)
+                )
+                .combined(with: .opacity)
+            )
+            .id("\(value)")
+    }
+
+    @ViewBuilder var decrementButton: StepperButton {
+        StepperButton(
+            isIncrement: false,
+            isEnabled: isEnabled ? value > minValue : false,
+            style: style,
+            clickAction: {
+                isIncreased = false
+
+                withAnimation {
+                    value -= 1
+                }
+            }
+        )
+    }
+
+    @ViewBuilder var incrementButton: StepperButton {
+        StepperButton(
+            isIncrement: true,
+            isEnabled: isEnabled ? value < maxValue : false,
+            style: style,
+            clickAction: {
+                isIncreased = true
+
+                withAnimation {
+                    value += 1
+                }
+            }
+        )
+    }
+
+    /// Creates Orbit Stepper component.
+    public init(
+        value: Binding<Int>,
+        minValue: Int,
+        maxValue: Int,
+        isEnabled: Bool = true,
+        style: Style
+    ) {
+        self._value = value
+        self.minValue = minValue
+        self.maxValue = maxValue
+        self.isEnabled = isEnabled
+        self.style = style
+    }
+}
+
+// MARK: - Subtypes
+extension Stepper {
+
+    struct StepperButton: View {
+
+        let isIncrement: Bool
+        let isEnabled: Bool
+        let style: Stepper.Style
+        let clickAction: () -> Void
+
+        var body: some View {
+            SwiftUI.Button(action: clickAction) {
+                ZStack {
+                    Circle()
+                        .fill(isEnabled ? style.backgroundActiveColor : style.backgroundColor)
+                        .frame(width: 24, height: 24)
+                    icon
+                }
+            }.disabled(!isEnabled)
+        }
+
+        var icon: Icon {
+            let color = isEnabled ? style.foregroundActiveColor : style.foregroundColor
+
+            return isIncrement ? .init(.plus, color: color) : .init(.minus, color: color)
+        }
+    }
+}
+
+// MARK: - Styles
+extension Stepper {
+
+    public enum Style {
+        case primary
+        case secondary
+
+        public var foregroundColor: Color {
+            switch self {
+                case .primary:                  return .white
+                case .secondary:                return .inkDark.opacity(0.5)
+            }
+        }
+
+        public var foregroundActiveColor: Color {
```

Comment:
The active colors seem off

<img width="190" alt="image" src="https://user-images.githubusercontent.com/35844477/216381870-fd3eb69c-fd38-4791-86cf-b3bc05348048.png">

Reply:
Should be better now, pls recheck, the off state now differs from active

Reply:
It still looks the same. Can you send a screenshot of how it should look?

Reply:
**Disabled** state example from Preview
<img width="114" alt="Screenshot 2023-02-09 at 13 56 34" src="https://user-images.githubusercontent.com/68693633/217820332-a15fcd53-af67-4a90-8ea7-31145dee1f7a.png">

**Disabled** state from Figma
<img width="112" alt="Screenshot 2023-02-09 at 13 57 37" src="https://user-images.githubusercontent.com/68693633/217820383-07575992-376f-4bd2-9eb5-86ea601e3181.png">

**Selected** state example from Preview
<img width="102" alt="Screenshot 2023-02-09 at 13 55 57" src="https://user-images.githubusercontent.com/68693633/217820455-ed08143f-86a8-42bf-b7dc-6f4658d3c08a.png">

Cannot find the **Selected** state in Figma, is it supposed to be a "Focus" one? P.S. I believe not
<img width="280" alt="Screenshot 2023-02-09 at 14 05 00" src="https://user-images.githubusercontent.com/68693633/217820826-31596931-6e4d-4036-ae5e-b570584921f5.png">

Reply:
It should be, because we currently do not support hover states (they would only be visible on iPad using pointing device), so the focus state is the one corresponding to "pressed"

Reply:
Add the pressed state:
<img width="134" alt="Screenshot 2023-02-10 at 16 20 10" src="https://user-images.githubusercontent.com/68693633/218128938-46b07941-4bd8-40e8-98fd-b659a616ada4.png">
<img width="118" alt="Screenshot 2023-02-10 at 16 20 13" src="https://user-images.githubusercontent.com/68693633/218128945-5f883b5b-f9f9-41c9-bca6-7427a6f16ac5.png">